src/util/tracking.ts

```diff
@@ -67,6 +72,7 @@ export interface TrackingValues extends LoginTrackingValues {
   sourcePluginId?: string // currency pluginId of dest asset
   numAccounts?: number // Number of full accounts saved on the device
   exchangeAmount?: number
+  nativeAmount?: number
```

Comment:
nativeAmount absolutely has to be a string. You will overflow otherwise with many currencies. While you're at it, change exchangeAmount back to a string as well for consistency with the rest of the app.